canonical-web-and-design/ubuntu.com
# Advantage Shop

⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️
### BEFORE MERGING / AFTER QA
Contract API url and Stripe public key must be updated to production values. They can be found [here](https://docs.google.com/document/d/1BG2mRWUF9pHAiwLPgm-7gelrZdstjlkQit84kXBum4A), and also here:
- contract API url: https://contracts.canonical.com
- Stripe public key: pk_live_68aXqowUeX574aGsVck8eiIE

⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️

# Done

Added the UA shop at /advantage/subscribe

# QA

- Check out this branch
- `dotrun` the site
- Visit /advantage/subscribe
- Purchases can be completed as either a logged in SSO user, or as a guest
- Make purchases, and try to break things
